modules/archiving-a-model.adoc

```diff
@@ -0,0 +1,34 @@
+:_module-type: PROCEDURE
+
+[id="archiving-a-model_{context}"]
+= Archiving a model
+
+[role='_abstract']
+You can archive a model. The model and all of its versions will be archived and unavailable for use unless it is restored.
```

Comment:
[Helpful to add] _You can archive a model that you no longer require_, or something similar.

Reply:
Updated.